examples/units/artist_demo.py

```diff
@@ -1,9 +1,9 @@
 """
 ============
-Artist tests
+Artist demo
 ============
 
-Test unit support with each of the Matplotlib primitive artist types.
+Demo unit support with each of the Matplotlib primitive artist types.
```

Comment:
Demonstrate?

(Again, not worth a modification on its own.)

Reply:
Fixed in rebase.